### PR TITLE
Remove bower

### LINKED ISSUE #919: Hudson build fails on Paper UI
Since https://github.com/eclipse/smarthome/pull/873 had been merged, the build on Hudson fails due to proxy problems, see https://github.com/eclipse/smarthome/pull/873:
```
[INFO] o.h.m.e.h.MavenExecutionResultHandler - [1] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.github.eirslett:frontend-maven-plugin:0.0.27:bower (bower install) on project org.eclipse.smarthome.ui.paper: Failed to run task
[DEBUG] Closing connection to remote
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:0.0.27:bower (bower install) on project org.eclipse.smarthome.ui.paper: Failed to run task: 'bower install --config.proxy=http://proxy.eclipse.org:9898' failed. (error code 1) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.github.eirslett:frontend-maven-plugin:0.0.27:bower (bower install) on project org.eclipse.smarthome.ui.paper: Failed to run task
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:120)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:355)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:155)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:584)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:216)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:160)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoFailureException: Failed to run task
	at com.github.eirslett.maven.plugins.frontend.mojo.AbstractFrontendMojo.execute(AbstractFrontendMojo.java:69)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:132)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	... 19 more
Caused by: com.github.eirslett.maven.plugins.frontend.lib.TaskRunnerException: 'bower install --config.proxy=http://proxy.eclipse.org:9898' failed. (error code 1)
	at com.github.eirslett.maven.plugins.frontend.lib.NodeTaskExecutor.execute(NodeTaskExecutor.java:59)
	at com.github.eirslett.maven.plugins.frontend.mojo.BowerMojo.execute(BowerMojo.java:42)
	at com.github.eirslett.maven.plugins.frontend.mojo.AbstractFrontendMojo.execute(AbstractFrontendMojo.java:67)
	... 21 more
```
@GradyD Could you please have a look?

### PR DESCRIPTION
**Pull request by GradyD**
This should fix #919 and accomplish #833

### CODE REVIEW

File: extensions/ui/org.eclipse.smarthome.ui.paper/package.json [hunk 1]
```diff
@@ -3,7 +3,14 @@
   "version": "0.8.0",
   "private": true,
   "dependencies": {
-    "bower": "^1.4.1",
+    "angular": "^1.4.9",
```
Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/package.json]: You should stick to the versions we had previously (1.3.13, 0.10.0 for angular-material, 3.3.2 for bootstrap, etc.) as these are the ones where we have IP approval.

**Comment by maggu2810** [comment]
Why does the PR "Remove bower" remove the file "web/partials/dialog.ruletemplate.html"?

**Comment by kaikreuzer** [comment]
I guess because it does not belong there - but @GradyD seems to have missed to add it to web-src instead...

**Comment by GradyD** [comment]
Looks like I might have mad 1 too many glasses of scotch last night.

I'll update the version numbers to match what was in bower. Should I hard match the versions or leave them loose upgrades (related to discussion on #801)?

The file "web/partials/dialog.ruletemplate.html" should have been moved to web-src. Nothing should be tracked inside of the /web/ folder. Ill do a move instead of remove so that the file will be tracked in the correct folder.

**Comment by kaikreuzer** [comment]
> Should I hard match the versions or leave them loose upgrades (related to discussion on #801)?

I would prefer switching to hard matches, since we have to make sure that we ship the approved versions.

**Comment by GradyD** [comment]
Sounds good. 

Is there a list or someplace I can look at what has been approved? Specifically want to make sure I hard match the approved version.

**Comment by kaikreuzer** [comment]
Well, the versions that were listed in the bower file are the approved ones.

**Comment by GradyD** [comment]
Cool, ill hard match off of those tonight and update this PR, thanks for the clarification.

**Comment by kaikreuzer** [comment]
Thanks!

**Comment by kaikreuzer** [comment]
Btw, I just tried the latest master, executed a "mvn clean install" on the Paper UI and started the runtime in the IDE - and everything is broken. It is lacking MANY files like
css/bootstrap.min.css
js/angular.min.js
and 8 others (within css and js).
Am I doing something wrong or is something missing in copying the files to the web folder?

**Comment by GradyD** [comment]
Latest master as in the current master branch or my master branch?

Can you tell if bower ran and installed correctly?

**Comment by kaikreuzer** [comment]
Hm, I thought it ran successful. I just cleaned everything again now and re-ran maven for the xth time and now all is there. So ignore my comment above for the moment...

**Comment by GradyD** [comment]
I've made all the needed improvements. I did run into a few issues,

* tinycolor2 is the equilivent of tinycolor bower file, https://github.com/bgrins/TinyColor#including-in-node
* Latest version 4.3 of roboto-fontface was never published to NPM, package no longer seems to be maintained. https://github.com/choffmeister/roboto-fontface-bower, choffmeister/roboto-fontface-bower#25
* angular-material 10.0 has a peer dependency on angular version 1.3.15 or higher. I moved to angular 1.3.15, was never an issue with bower because we were not hard referencing versions.
* angular-animate was never included in the bower file but is a peer dependency of angular-material. angular-animate is being used in the app
* angular-aria was never included in the bower file but is a peer dependency of angular-material. angular-aria is being used in the app

My guess is with the two peer dependencies is that bower auto installed them, NPM auto installed them.

**Comment by kaikreuzer** [comment]
You are right, I just checked, the very same files and versions were already included in the past.
So we indeed have to sort out the right versions (we might do this in the context of upgrading them anyhow), but this is unrelated to this PR, so I am fine with merging it.

**Pull request merged**